src/main/static/assets/templates/page_home.jade

```diff
@@ -32,14 +32,17 @@ microprofileio-header.home
   .card.text-image.flipped.pdf
     div
       div
-        h2 Eclipse MicroProfile 1.3 is now available!
-        p Version 1.3 consists of MicroProfile 1.2 plus the following updates: Config 1.2, Metrics 1.1, OpenAPI 1.0, OpenTracing 1.0, REST Client 1.0.
-        p Download the specification PDF to learn more about the architecture and required APIs.
-        a(href="https://github.com/eclipse/microprofile-bom/releases/download/1.3/microprofile-spec-1.3.pdf", target="_blank") Download PDF
+        h2 Eclipse MicroProfile 1.4 & 2.0 is now available!
+        h3 MicroProfile now offers the same level of functionality to be usable with either Java EE 7 or Java EE 8.
+        p Eclipse MicroProfile 1.4, which builds on the 1.3 version, updates the Config, JWT Propagation, Fault Tolerance, Open Tracing, and Rest Client APIs.        
+        a(href="https://github.com/eclipse/microprofile/releases/download/1.4/microprofile-spec-1.4.pdf", target="_blank") Download 1.4 PDF
+        p <br />
+        p Eclipse MicroProfile 2.0, which builds on the 1.4 version, updates the CDI, JSON-P, JAX-RS APIs to their Java EE 8 equivalents, and adds the JSON-B APIs.
```

Comment:
"JSON-B APIs" should read "JSON-B API" (singular).

Reply:
Thank you @mikecroft , I opened https://github.com/microprofile/microprofile-site/pull/79 for this issue.